Weather/Weather/Controllers/NewCityController.swift

```diff
@@ -0,0 +1,48 @@
+//
+//  NewCityController.swift
+//  Weather
+//
+//  Created by Oleg Yankiwskyi on 5/16/18.
+//  Copyright © 2018 Oleg Yankiwskyi. All rights reserved.
+//
+
+import UIKit
+
+class NewCityController: UIViewController {
+
+    @IBOutlet weak var cityTextField: UITextField!
+    let citiesModel = CitiesModel()
```

Comment:
Critical error: you create a new model for the controller. New city will never appear in cities list.
`NewCityController` should notify its delegate about new city - via closure, delegate method or notification